Fix Vale linter errors in config-intro.adoc (#10195)
circleci/circleci-docs: docs/guides/modules/getting-started/pages/config-intro.adoc

+
* *Line 6:* The `steps` collection is a list of `run` directives.
* *Line 7:* Each `run` directive is executed in the order in which it is declared.

> * *Line 8:* The `name` attribute provides useful information when returning warnings, errors, and output. The `name` must be meaningful to you as an action within your build process.

Author: ```suggestion
* *Line 8:* The `name` attribute provides useful information when returning warnings, errors, and output. It is helpful if the `name` is meaningful to you as an action within your build process.
```
